src/helpers/log.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+import path from 'path'
+import rimraf from 'rimraf'
+import resolveUserDataDirectory from './resolveUserDataDirectory'
+
+export const resolveLogsDirectory = () => {
+  const { LEDGER_LOGS_DIRECTORY } = process.env
+  if (LEDGER_LOGS_DIRECTORY) return LEDGER_LOGS_DIRECTORY
+  const electron = require('electron')
+  return path.resolve((electron.app || electron.remote.app).getPath('userData'), 'logs')
+}
+
+export const cleanUpBeforeClosingSync = () => {
+  rimraf.sync(resolveLogsDirectory(), { disableGlob: true })
+  rimraf.sync(path.resolve(resolveUserDataDirectory(), 'sqlite/*.log'))
```

Comment:
not a big deal but I'm curious about why we can't do it "async" and we run both rimraf in parallel

Reply:
we'll keep like this for now, not such a big deal because it's at the time you press the X (or CMD+Q) and we somehow want to block the "close" event